### ISSUE TITLE
XamlUIBasics sample does not compile against latest (16267) Preview SDK

### ISSUE THREAD

**Issue by asimonov-msft** [Microsoft/Windows-universal-samples#715]
The sample project fails to compile with the following errors, due to recent [removal of TreeView from the SDK](https://blogs.windows.com/buildingapps/2017/08/08/windows-10-sdk-preview-build-16257-mobile-emulator-build-15235-released/#tWCyBWoymwLxv2Zj.97):
```
D:\dev\Windows-universal-samples\Samples\XamlUIBasics\cs\AppUIBasics\ControlPages\TreeViewPage.xaml.cs(13,9,13,21): error CS0246: The type or namespace name 'TreeViewNode' could not be found (are you missing a using directive or an assembly reference?)
D:\dev\Windows-universal-samples\Samples\XamlUIBasics\cs\AppUIBasics\obj\x86\Debug\ControlPages\TreeViewPage.g.i.cs(20,50,20,58): error CS0234: The type or namespace name 'TreeView' does not exist in the namespace 'Windows.UI.Xaml.Controls' (are you missing an assembly reference?)
```

**Comment by andrejo-msft** [comment]
#741 should take care of the problem.

**Comment by oldnewthing** [comment]
Fixed in v5.0.3.
